Resources/input/fsh/codesystems/CancellationReasonTypeCS.fsh

```diff
@@ -0,0 +1,8 @@
+CodeSystem: CancellationReasonTypeCS
+Id: CancellationReasonTypeCS
+Title: "CodeSystem for CancellationReasonType"
+Description: "Definition of reasons why a prescriptionRequest was cancelled"
+* insert Meta
+
+* #informationMissing "Fehlende Informationen"
+* #informationenIncorrect "Fehlerhafte Informationen"
```

Comment:
Mir fällt auch kein Beispiel ein aber ließe sich ein Wort finden, was sich mehr voneinander unterscheidet? Fehlend und Fehlerhaft sind sehr nah beieinander. In einem Dropdown evtl einfacher wenn die sich mehr unterscheiden würden.

Vielleicht Abhandene Informationen?

Reply:
Geändert in
* "Informationen unvollständig"
* "Informationen fehlerhaft"